src/app/app.html

```diff
@@ -34,6 +34,12 @@ <h3>CodeMirror Mode PromQL</h3>
   <option selected value="offline">Offline</option>
   <option value="prometheus">Prometheus</option>
 </select>
+<br>
+<label for="fullLanguage">Language to complete</label>
```

Comment:
I guess this and the line below now also shouldn't say `fullLanguage` / `fullPromQLLanguage` anymore, but something like `languageType`.